src/iexamine.cpp

```diff
@@ -2357,8 +2357,9 @@ void iexamine::arcfurnace_empty( player &p, const tripoint &examp )
         } else if( i.made_of_any( arcfurnaceable ) ) {
             fuel_present = true;
         } else {
-            add_msg( m_bad, _( "This furnace contains %s, which can't be made into calcium carbide!" ), i.tname( 1,
-                     false ) );
+            add_msg( m_bad, _( "This furnace contains %s, which can't be made into calcium carbide!" ),
+                     i.tname( 1,
+                              false ) );
```

Comment:
You can collapse these onto one line now.

Reply:
I dunno, this is what astyle spat out.

Reply:
Yeah, astyle doesn't remove line breaks, and this was previously separated. Astyle won't complain if you collapse the above to lines into one, like this
```cpp
i.tname( 1, false ) );
```